cmd/query/app/apiv3/otlp_translator.go

```diff
@@ -0,0 +1,364 @@
+// Copyright (c) 2021 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiv3
+
+import (
+	"encoding/binary"
+	"fmt"
+	"hash/fnv"
+	"math"
+	"sort"
+	"strconv"
+	"strings"
+
+	"go.opentelemetry.io/collector/translator/conventions"
+	tracetranslator "go.opentelemetry.io/collector/translator/trace"
+
+	"github.com/jaegertracing/jaeger/model"
+	commonv1 "github.com/jaegertracing/jaeger/proto-gen/otel/common/v1"
+	resourcev1 "github.com/jaegertracing/jaeger/proto-gen/otel/resource/v1"
+	v1 "github.com/jaegertracing/jaeger/proto-gen/otel/trace/v1"
```

Comment:
related to https://github.com/jaegertracing/jaeger/pull/2988#discussion_r632994232, i think something like `otlptrace.ResourceSpans` is more readable than `v1.ResourceSpans`. Perhaps the same for `commonv1` and `resourcev1`.

likewise, I think it would be easier to read `jaeger.Span` than `model.Span`.

Reply:
I have renamed it to `v1` to `tracev1`